src/components/checkBox/checkBox.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import CheckBox from ".";
+
+
+export default {
+  title: "CheckBox",
+  component: "CheckBox"
+};
+
+
+
+
+export const defaultCheckBox = () => <CheckBox/>;
+
+export const CheckBoxChecked = () => <CheckBox checked ></CheckBox>;
+
+export const disabledCheckBox = () => <CheckBox disabled style={{ backgroundColor: "#F0F0F0" }}></CheckBox>;
+
+export const checkedDisabledCheckBox = () => <CheckBox default checked disabled style={{ backgroundColor: "#F0F0F0" }} ></CheckBox>;
```

Comment:
I don't think there should be a `checkedDisabled` state, a disabled state communicates when a component or element isn't interactive and should be deemphasized in the UI hence a disabled checkbox should not be checked.

Reply:
I just did that based on the style guide

Reply:
but it's removed now